client/sites/settings/web-server/defensive-mode-form.tsx

```diff
@@ -66,14 +61,14 @@ export default function DefensiveModeForm( {
 	const enabledUntil = moment.unix( defensiveModeData?.enabled_until ?? 0 ).local();
 
 	return (
-		<ContainerComponent
-			className="defensive-mode-card"
-			title={ translate( 'Defensive mode', {
-				comment: 'Defensive mode is a feature to protect against DDoS attacks.',
-				textOnly: true,
-			} ) }
-		>
-			<DescriptionComponent>
+		<PanelSection>
+			<PanelHeading>
```

Comment:
Do we need to pass an `id` here as well, just like [here](https://github.com/Automattic/wp-calypso/pull/96275/files#diff-eee737ddca1b99b1765c62abc1d2852e6f76cfacce8d9ac96e85649d5709cab8R429)?

Reply:
No, the id was required in your mentioned link because we have a link pointing to that exact card. It's not the case for this Defensive mode card.